src/main/java/Parser.java

```diff
@@ -0,0 +1,36 @@
+public class Parser {
+    protected String userInput;
+    
+    public Parser () {}
+    
+    // no error checking for processing task information has been implemented (e.g. deadline but no /by)
+    public String[] processTaskInformation(String userInput) {
```

Comment:
I like that you have a separate class to parse information, makes reading code a lot easier!